source/_docs/configuration/securing.markdown

```diff
@@ -3,14 +3,44 @@ title: "Securing"
 description: "Instructions on how to secure your Home Assistant installation."
 ---
 
-One major advantage of Home Assistant is that it's not dependent on cloud services. Even if you're only using Home Assistant on a local network, you should take steps to secure your instance.
+One major advantage of Home Assistant is that it is not dependent on cloud services.  By default, you are in control of your data. The decisions you make will affect your security and your privacy.  We take your privacy and security seriously.  Because of this, even if you are only using Home Assistant on a local network, you should take steps to secure your instance.
 
-## Checklist
+<div class='info'>
+Disclaimer: Home Assistant is not your security advisor.  Due to continually changing threat landscapes and individual configurations we recommend best practices.  The advice given here should be considered as a security baseline and should not represent your entire security profile.  
+</div>
 
-Here's the summary of what you *must* do to secure your Home Assistant system:
+### Initial Setup
+
+After installation we strongly recommend you perform the following tasks before using the system.  
+
+| Task | Justification |
+| --- | --- |
+| Setup [encrypted remote access](/docs/configuration/securing/#remote-access). | Setup HTTPS to encrypt your communication.  HTTP will allow eavesdropping on your passwords. |
+| Secure [access methods](docs/configuration/securing/#extras-for-manual-installations) | Certain installation methods have dependencies on SSH and other access methods. It is best to remove or secure these. |
```

Comment:
I think we should discourage the use of remote access. Writing it like so, might people look into this while they might even not need it. From previous experience on similar texts, we know people will do this because it is written here, regardless if it applies to them or not.

```suggestion
| Set up [encrypted remote access](/docs/configuration/securing/#remote-access). | Setup HTTPS to encrypt your communication. HTTP will allow eavesdropping on your passwords. |
| Secure [access methods](docs/configuration/securing/#extras-for-manual-installations) | Certain installation methods have dependencies on SSH and other access methods. It is best to remove or secure these. |
```

Reply:
The remote access instructions refer to Nabu Casa and other items on the same page.  Maybe we should look at elaboration in that section to ensure the user understands risk vs rewards.   Or we can always simply drop the "remote" verbiage to set up encrypted access.  It's recommended to use encryption even if it's on your LAN.

Reply:
> It's recommended to use encryption even if it's on your LAN.

I'm voting that out of scope for our documentation. It will cause a magnitude of problems most people can't deal with. I won't allow writing documentation that will drive people into issues we have to deal with as a project.